### DISCUSSION TITLE
Is there a way to send telemetry data to OTEL collector over UDP?

### DISCUSSION THREAD

**Discussion by matheus-artero** [open-telemetry/opentelemetry-collector#9979] [Q&A]
I am trying to use OTEL collector with udp to avoid a large number of simultaneously tcp connections

**Comment by atoulme** [comment]
I believe the statsd receiver supports UDP. https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/carbonreceiver/README.md
